deals/options.go

```diff
@@ -0,0 +1,75 @@
+package deals
+
+import "os"
+
+// Config contains configuration for storing deals.
+type Config struct {
+	ImportPath string
+}
+
+// Option sets values on a Config.
+type Option func(*Config) error
+
+// WithImportPath indicates the import path that will be used
+// to store data to later be imported to Lotus.
+func WithImportPath(path string) Option {
+	return func(c *Config) error {
+		if err := os.MkdirAll(path, 0700); err != nil {
+			return err
+		}
+		c.ImportPath = path
+		return nil
+	}
+}
+
+// ListDealRecordsConfig specifies the options for DealsManager.List.
+type ListDealRecordsConfig struct {
```

Comment:
Config and options for listing storage deals

Reply:
Both storage and retrieval record funcs use the same options. A couple options don't apply to retrieval deals, and that is noted in the code docs